plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -99,6 +100,16 @@ func RegisterFitPredicate(name string, predicate algorithm.FitPredicate) string
 	return RegisterFitPredicateFactory(name, func(PluginFactoryArgs) algorithm.FitPredicate { return predicate })
 }
 
+// RegisterRequiredFitPredicate registers a fit predicate with the algorithm registry,
+// it's always included in configuration. Returns the name with which the predicate was registered.
+func RegisterRequiredFitPredicate(name string, predicate algorithm.FitPredicate) string {
```

Comment:
I don't like this - it should be a normal predicate, exactly as fit resources and stuff. Is someone want to break their cluster they're free to do so.